pyMBE.py

```diff
@@ -450,15 +455,14 @@ def center_molecule_in_simulation_box(self, molecule_id, espresso_system):
             molecule_id(`int`): Id of the molecule to be centered.
             espresso_system(`obj`): Instance of a system object from the espressomd library.
         """
-        center_of_mass = self.calculate_center_of_mass_of_molecule ( molecule_id=molecule_id,espresso_system=espresso_system)
-        box_center = [espresso_system.box_l[0]/2.0]*3
-        pmb_type = self.df.loc[self.df['molecule_id']==molecule_id].pmb_type.values[0]
-        pmb_objects = ['protein','molecule','peptide']
-        if pmb_type in pmb_objects:
```

Comment:
You are right that this check is not really necessary, However, what is missing is a sanity check to ensure that the input molecule_id is defined in the pmb.df, otherwise the user will get a somewhat confusing pandas error

Reply:
I have added such a check.